mfr/extensions/audio/templates/viewer.mako

```diff
@@ -3,5 +3,4 @@
     Your browser does not support the audio tag.
 </audio>
 
-<script src="/static/js/mfr.js"></script>
-<script src="/static/js/mfr.child.js"></script>
+<script src="/assets/mfr.child.js"></script>
```

Comment:
Only reason mfr.js was included prior was to get pym.js library in the iframe. Webpack builds it in now.